datacats/cli/pull.py

```diff
@@ -22,9 +25,16 @@ def pull(opts):
     """Download or update required datacats docker images
 
 Usage:
-  datacats pull
+  datacats pull [-d]
+
+Options:
+  -d --devel         Pull extra "development" images. These
```

Comment:
I like `--all`
"Pull optional images as well as required images. Optional images will be pulled when needed, ..."